test/e2e/auth/dp/dp_auth_universal_test.go

```diff
@@ -0,0 +1,9 @@
+package auth_test
+
+import (
+	. "github.com/onsi/ginkgo"
+
+	"github.com/kumahq/kuma/test/e2e/auth/dp"
+)
+
+var _ = Describe("Test Universal", auth.DpAuthUniversal)
```

Comment:
Suggest putting this in `auth_dp_suite_test.go` so there are fewer places to go looking for things.

Reply:
make sense, moved

Reply:
+1 on this. Does anyone mind me opening a PR to move all these tests files in _suite_test?